### DISCUSSION TITLE
Do I have to reload mergerfs folder if I add a new one?

### DISCUSSION THREAD

**Discussion by ghost** [trapexit/mergerfs#1076] [Q&A] [answered]
I have a mergerfs mount
```
$ sudo cat /etc/fstab
/mnt/hdd*                                 /mnt/storage fuse.mergerfs allow_other,use_ino,cache.files=off,dropcacheonclose=true,ignorepponrename=true,func.mkdir=epall,x-gvfs-show   0       0
```

I've mount another folder with sshfs
```
$ sudo mkdir -p /mnt/hdd0
$ sudo chown user:user /mnt/hdd0
$ sshfs user@host:/mnt/hdd0 /mnt/hdd0
$ ls /mnt/hdd0
Audio  Backups  Downloads  Games  lost+found  Software
```

I was expecting mergerfs to use the new folder too, but it doesn't. If I need to, how do I reload the merged folder?

PS: as a reference, [see this](https://unix.stackexchange.com/a/366778) to deal with server disconnection while using sshfs.

**Comment by trapexit** [comment]
https://github.com/trapexit/mergerfs#globbing

> NOTE: the globbing is done at mount or when updated using the runtime API. If a new directory is added matching the glob after the fact it will not be automatically included.

**Comment by trapexit** [comment, marked as answer]
You have to remount or use the runtime API to tell mergerfs about the new branch.
